ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -41,6 +41,16 @@ tests:
     - ref: baremetalds-devscripts-conf-dualstack
     - ref: baremetalds-devscripts-setup
     workflow: baremetalds-e2e
+- as: e2e-metal-ipi-dualstack-periodic
+  cron: 0 */8 * * *
```

Comment:
`*/8` means 'every 8 hours' I guess? Maybe just make it once a day for now and we can update it later

Reply:
Done